internal/platform/platform.go

```diff
@@ -93,14 +93,20 @@ func init() {
 		name:  "openstack",
 		fetch: openstack.FetchConfig,
 	})
+	configs.Register(Config{
+		name:       "aws",
+		fetch:      aws.FetchConfig,
+		newFetcher: aws.NewFetcher,
+	})
+	// FIXME: compatibility alias; delete after a transition period
 	configs.Register(Config{
 		name:       "ec2",
-		fetch:      ec2.FetchConfig,
-		newFetcher: ec2.NewFetcher,
+		fetch:      aws.FetchConfig,
+		newFetcher: aws.NewFetcher,
 	})
 	configs.Register(Config{
-		name:  "gce",
-		fetch: gce.FetchConfig,
+		name:  "gcp",
```

Comment:
Should we have a compatibility alias similar to the `ec2` alias for `aws` under the same transition period?

Reply:
This change only affects spec 3.x, and FCOS/RHCOS don't support GCP yet, so there shouldn't be anything to be compatible with.